### ISSUE TITLE
add extant ID to getMPInfo function example

### ISSUE THREAD

**Issue by davewhiteland** [mysociety/theyworkforyou#789]
http://www.theyworkforyou.com/api/docs/getMPInfo doesn't document the returned fields, but it's easy to see what you get back by making a call... but to do that you have to look up an ID.

I suggest we just pre-populate it with an existing ID so informative, useful results will be returned with one click from that page.

(Maybe similar applies to similar pages with demo calls on them)

**Comment by tmtmtmtm** [comment]
@davewhiteland do you have a suggestion for a good ID to use for this?

**Comment by struan** [comment]
I don't think it matters really. I would either pick 10001 which is Dianne Abbot or 25386 which is Daniel Zeichner simply as they are alphabetically first or last and that's as good an arbitrary means of choosing as any.
